### ISSUE TITLE
when trying to type in time value date field get empty

### ISSUE THREAD

**Issue by deli6z** [mgcrea/angular-strap#1600]
When using datepicker and timepicker for one ngmodel field after date was selected we need to support manual type in of time, but it clears out the date field once start typing.
Also if you leave the date field empty and select time first then date field defaults to 1/1/1970. Is there a way to set it to today if time is set first? 

Thank you

**Comment by vmlf01** [comment]
The ngmodel value is getting invalidated when you type the time part.

**Comment by adam-marshall** [comment]
Agree, the first issue is terrible but hard to avoid. Second is annoying.

**Comment by sfichera** [comment]
Hi, 

@deli6z, @vmlf01, @adam-marshall  
Have you found a workaround in order to avoid this issue? 

Thanks & Regards!!
